videos-list/11-mo.md

```diff
@@ -14,8 +14,15 @@ https://discourse.pymc.io/t/a-novel-bayesian-model-to-fit-spectrophotometric-dat
 
 ## Timestamps
 - 0:00 Start of event
-- x:xx 
-- x:xx
+- 2:03 Observable Astronomy:  Grism Spectroscopy
+- 3:42 Understanding Quiescent galaxies:  How do they form so quickly?
+- 5:35 Physical Model
+- 8:16 Build the model - spectroscopy
+- 11:27 Build the model - HST photometry
+- 15:15 Build the model - Spitzer photometry
+- 12:52 Build the model - Nuisance Parameters
+- 20:50 Test prior using mock data
+- 30:00 Summary
```

Comment:
```suggestion
02:03 Observable Astronomy:  Grism Spectroscopy
03:42 Understanding Quiescent galaxies:  How do they form so quickly?
05:35 Physical Model
08:16 Build the model - spectroscopy
11:27 Build the model - HST photometry
15:15 Build the model - Spitzer photometry
12:52 Build the model - Nuisance Parameters
20:50 Test prior using mock data
30:00 Summary
```

Reply:
@zbraiterman Can you check the "15:15" timestamp here, since it's listed here before the "12:52" timestamp?
```
11:27 Build the model - HST photometry
15:15 Build the model - Spitzer photometry
12:52 Build the model - Nuisance Parameters
```